extension/Makefile

```diff
@@ -76,6 +79,18 @@ install-pgmq:
 	PG_CONFIG=${PGRX_PG_CONFIG} make install && \
 	cd .. && rm -rf pgmq
 
+install-vectorscale:
+	@ARCH=$$(uname -m); \
+	if [ "$$ARCH" != "arm64" ]; then \
+		export RUSTFLAGS="-C target-feature=+avx2,+fma"; \
+	else \
+		echo "Running on arm64, no variable set"; \
```

Comment:
What does "no variable set" mean here? Could this message be made clearer/more specific?

Reply:
I'll change that to say something like ~not setting cpu instruction flags